### ISSUE TITLE
No more S20 on aliexpress - B25 support?

### ISSUE THREAD

**Issue by markbosshard** [cherezov/orvibo#14]
Hi

As there is no more EU S20 on aliexpress, do you plan to support the successor B25 soon? Thanks.

**Comment by BikeTech** [comment]
Yes, it would be great to be able to manipulate the S25 here in the US from code!
